src/Tests/UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -36,7 +36,8 @@
 
 #if NET5_0
 // Test SupportedOSPlatform warnings for APIs targeting 10.0.19041.0:
-[assembly: global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.18362.0")]
+//[assembly: global::System.Runtime.Versioning.SupportedOSPlatform("Windows")]
```

Comment:
Attribute seems the same as below, intentional?

Reply:
nope - unintentional